src/shared/lib/validation/proposal/sprint-with-us.ts

```diff
@@ -83,34 +85,47 @@ export function validateSWUProposalReferences(raw: any): ArrayValidation<CreateS
   return validateArrayCustom(raw, validateSWUProposalReference, {});
 }
 
-// TODO validate word limit
 export function validateSWUProposalTeamQuestionResponseResponse(raw: string, wordLimit = MAX_TEAM_QUESTION_WORD_LIMIT): Validation<string> {
-  return validateGenericString(raw, 'Response', 1);
+  if (count(raw, 'words', {}) > wordLimit) {
```

Comment:
Also, could you move this to a helper like `countWords` in `shared/lib/index.ts`? Also, we should have a generic validation function like `validateGenericStringWords` to validate by word count instead of char count, and use it here.

Reply:
Will do.

Reply:
TY